app/Kernel.php

```diff
@@ -32,6 +36,8 @@
      */
     public function __construct($environment, $debug)
     {
+        $this->request = Request::createFromGlobals();
```

Comment:
normally it would, but we need the request in 2 different functions, and one of those always runs, that's why I added it there so I wouldn't need to do Request::createFromGlobals twice